diagnostic_aggregator/src/add_analyzer.cpp

```diff
@@ -0,0 +1,112 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2009, Willow Garage, Inc.
```

Comment:
This should be the BV where you work for, so `Copyright 2023 Nobleo Technology B.V.` but let me check with Rokus

Reply:
Wrote it myself but just copied the pasta for the notice and forgot to make it Nobleo Technology or whatever it should be, will update.